example-application/data-access/config/config.js

```diff
@@ -1,11 +1,11 @@
 module.exports = {
-  username: 'myuser',
-  password: 'myuserpassword',
-  database: 'shop',
-  host: 'localhost',
-  port: 54320,
+  username: "myuser",
+  password: "myuserpassword",
+  database: "shop",
+  host: "localhost",
+  port: 54310,
```

Comment:
why?

Reply:
Another project in my computer is already using 54320/5432